src/app/app.page/app.page.component.ts

```diff
@@ -392,8 +392,15 @@ export class AppPageComponent implements OnInit, OnDestroy, AfterViewInit {
 
             this.restService.isEnterpriseEdition().subscribe(data => {
                 this.isEnterpriseEdition = data["success"];
-            });
 
+                if(this.isEnterpriseEdition){
+                    this.enterpriseEditionText = "";
+                }
+                else{
+                    this.enterpriseEditionText = "<div class=\" text-center isEnterprise\" ><label>Get <a target=\"_blank\" href=\"https://antmedia.io/#contact\">Enterprise Edition</a> for Ultra Low Latency Streaming</label></div>";
+                }
+
+            });
```

Comment:
Don't make the text reside in the code. 
Make the code has only flag(true/false) and let the UI behave accordingly.